_dashboards/dev-tools/run-queries.md

```diff
@@ -9,44 +9,36 @@ redirect_from:
-
-## Sending the request 
-
-To send a query to OpenSearch, select the query by placing the cursor anywhere in the query text. Then choose the play icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/play-icon.png" class="inline-icon" alt="play icon"/>{:/}) on the upper right of the request or press `Ctrl/Cmd+Enter`:
-
-<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/dev-tools-send.png" alt="Send request">
-
-OpenSearch displays the response in the response pane on the right side of the console:
-
-<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/dev-tools-response.png" alt="Response pane">{: .img-fluid }
+Feature | Description | How to use |
+--------|-------------|------------|
+**Collapsing or expanding a query** | Helpful to use when you have a long query or only want to focus on a specific part of the query. | To hide or show details of your query, select the expander arrow ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/arrow-down-icon.png" class="inline-icon" alt="arrow down icon"/>{:/}) next to the line number. |
+**OpenSearch search language** | Helpful to use when you need flexibility in writing complex queries. | To learn how to use the OpenSearch query domain-specific language (DSL), see [Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/). |
+**Single-line comments** | Helpful to use when you want to explain what the query does or to exclude parts of a query. | To add notes that are not interpreted by OpenSearch to your queries, use a hash (`#`) to start the single-line comment. |
+**Autocomplete** | Helpful to use when for suggestions for correct syntax and keywords, avoiding typos and errors, and reducing the amount of typing you do. | To define your preferences for autocomplete suggestions, configure them in **Settings**. |
+**Auto indenting** | Helpful to use to put the request body on a single line, which is useful for working with [bulk APIs]({{site.url}}{{site.baseurl}}/api-reference/document-apis/bulk/). Auto indenting a collapsed query expands it. | To use auto indent, select the queries that you want to format, then select the wrench icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/wrench-icon.png" class="inline-icon" alt="wrench icon"/>{:/}), and choose **Auto indent**. |
+**Request history** | Helpful for displaying up to the 500 most recent requests that OpenSearch ran successfully. | To view request history, select **History** from the top menu. If you select the request you want to view from the left pane, the query is shown in the right pane. To copy the query into the editor pane, select the query text and then select **Apply**. To clear the history, select **Clear**. |
+**Keyboard shortcuts** | Helpful for saving time, improving efficiency, and avoiding typos and errors. | To view all available keyboard shortcuts, select **Help** from the top menu. |
```

Comment:
Some of these descriptions are probably self-explanatory. I would remove the descriptions and just list what the command does in "how to use".